modules/getting-started/proc-audit-log-file-rotation.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+
+[id="proc-audit-log-file-rotation_{context}"]
+= Configure Audit Log file rotation
+
+You can enable file rotation for audit logs in {product}, and configure the file log location, name format, log file size, and archiving.
+
+.Prerequisite
+
+* You are an administrator of {product-short}.
+
+.Procedure
+
+. Enable audit logging to a rotating file:
++
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    enabled: true
+----
++
+. Customize the log file location:
++
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    logFileDirPath: /custom-path <1>
+----
+<1> An absolute path is recommended.
++
+[NOTE]
+====
+The specified directory will be created automatically if it does not exist.
+====
++
+. Customize the log file name format:
++
+.Example default log file name format
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    logFileName: custom-audit-log-%DATE%.log
+----
++
+. Configure file rotation frequency:
++
+.Example of file rotation frequency
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    frequency: '12h' # Default: `custom`
+    dateFormat: 'YYYY-MM-DD' # Default: `YYYY-MM-DD`
+    utc: false # Default: `false` #<1>
+    maxSize: 100m # Default: undefined
+----
+<1> Set to `utc` to `true` to use UTC time for `dateFormat` instead of local time.
++
+[NOTE]
+====
+The default file rotation occurs daily at 00:00 local time.
+====
++
+. Set a maximum log file size:
++
+.Example of maximum log file size
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    maxSize: 100m # Sets a max file size limit of 100MB for audit log
+----
+
+. Configure a retention policy by the number of files to keep or by the number of days to retain:
++
+.Example yaml number of files
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    maxFilesOrDays: 14 # Deletes the oldest log when there are more than 14 log files
+----
++
+.Example yaml number of days
+[source, yaml]
+----
+auditLog:
+  rotateFile:
+    maxFilesOrDays: 5d # Deletes logs older than 5 days
+----
++
+[NOTE]
+====
+* If log deletion is enabled, a `.<sha256 hash>-audit.json` is generated in the directory where the logs are to track generated logs. Any log file not contained in the directory is not subject to automatic deletion.
```

Comment:
I'd suggest adding a context statement explaining the note before the bullet points.